client/js/recurring_donations/edit/amount-step.es6

```diff
@@ -69,9 +69,8 @@ function chooseNewDonationAmount() {
 // All the buttons and the custom input for the amounts to select
 function amountFields(state) {
     if(state.params$().single_amount) return ''
-    return h('div.u-inline.fieldsetLayout--three--evenPadding', [
-        h('span',
-            R.map(
+    return h('div.fieldset-grid', [
```

Comment:
can we remove the wrapping array here?

For example, is there a reason that 


```javascript
[
   ...R.map(...., someArray)
]
```

can't be simplified to `R.map(...., someArray)`?

Reply:
Basically there are other things after the mapped items within the same array, so with the new CSS it's going from 
```js
[
   span with R.map(...., someArray) as its children array,
   custom amount,
   next button
]
```
to
```js
[
    ...R.map(...., someArray),
	custom amount,
	next button
]
```
putting the numbers and the other items all within one grid parent

Reply:
Oops, I missed those. Thanks!